OpenRA.Mods.Cnc/Traits/Mine.cs

```diff
@@ -41,7 +41,7 @@ void INotifyCrushed.OnCrush(Actor self, Actor crusher, BitSet<CrushClass> crushC
 			if (!info.CrushClasses.Overlaps(crushClasses))
 				return;
 
-			if (crusher.Info.HasTraitInfo<MineImmuneInfo>() || (self.Owner.Stances[crusher.Owner] == PlayerRelationship.Ally && info.AvoidFriendly))
+			if (crusher.Info.HasTraitInfo<MineImmuneInfo>() || (crusher.Owner.RelationshipWith(self.Owner) == PlayerRelationship.Ally && info.AvoidFriendly))
```

Comment:
Are you sure that inverting the players here is correct? The original logic (if the mine considers the crusher to be an ally, ignore it) made sense to me.

Reply:
I think you're right, it doesn't make sense in this case. Will revert. (I believe the idea was that the crusher decides which actor to crush, which makes sense for infantry and crates, but not for mines.)